tests/test_cases.md

```diff
@@ -36,11 +36,14 @@ mozdownload --url=https://mozqa.com/index.html -d url-downloads
 mozdownload --url=https://mozqa.com/index.html --username=username --password=password
```

Comment:
Sorry to pretend to be silly, but which tests exactly?

Retries, I just put in, so you can't mean those. The password username one? All the error ones?

Again sorry, just making sure that I don't have to revert out of a patch because I misunderstood